media-gfx/netgen/metadata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>sci-mathematics@gentoo.org</email>
+		<name>Gentoo Mathematics Project</name>
+	</maintainer>
+	<maintainer type="person" proxied="yes">
+		<email>waebbl-gentoo@posteo.net</email>
+		<name>Bernd Waibel</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Gentoo Proxy Maintainers</name>
+	</maintainer>
```

Comment:
Since this is an import from the ::science overlay we can list the science project as the proxy:
```suggestion
	<maintainer type="project" proxied="proxy">
		<email>sci-mathematics@gentoo.org</email>
		<name>Gentoo Mathematics Project</name>
	</maintainer>
	<maintainer type="person" proxied="yes">
		<email>waebbl-gentoo@posteo.net</email>
		<name>Bernd Waibel</name>
	</maintainer>
```

Reply:
same for the proxy maintainers project, I guess

Reply:
> same for the proxy maintainers project, I guess

Yes, but if we list proxy-maint as maintainer as well than we need ACK from @gentoo/proxy-maint

Reply:
Doesn't proxy-maint have to be added any time, because I'm a proxy-maintainer?

Reply:
> Doesn't proxy-maint have to be added any time, because I'm a proxy-maintainer?

Not necessarily, there needs to be a proxy, but the proxy does not have to be the proxy-maint project. Any developer/project can be a proxy.

Reply:
Oh I see, didn't know this. I always assumed, for a proxy maintainer, proxy-maint does always need to be listed as a proxy. Then I will drop the proxy-maint, for ease of use. If someone later expects it to be there, it can always get added again.